mobile/src/main/res/values/styles.xml

```diff
@@ -11,4 +11,23 @@
     <style name="WhiteText">
         <item name="android:textColor">#ffffff</item>
     </style>
+
+    <style name="openHAB.Dialog.Base" parent="@style/Theme.MaterialComponents.DayNight.Dialog.Alert">
+        <item name="dialogCornerRadius">16dp</item>
```

Comment:
Where does that 16dp figure come from? IMHO it's a bit much, 8dp would be the value I'd choose.

Other than that, LGTM.

Reply:
It comes from https://material.io/components/dialogs#theming 
I can have a look how to looks with 8dp.

Reply:
8dp looks also ok, I reduced the corner size.